ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -781,6 +781,21 @@ tests:
     cluster_profile: packet-assisted
     workflow: agent-e2e-appliance-ipv4
   timeout: 6h0m0s
+- as: e2e-agent-compact-ipv4-add-nodes
+  cluster: build05
```

Comment:
Probably for the first period I'd better to have it optional, ie: 

```
optional: true
```

Reply:
Done.